### PR TITLE
Post-release hotfix!

### PR DESCRIPTION
**Pull request by ChristianLieven**
Fixes issue with clashing dependencies and `memote online` not working on the first try!

### CODE REVIEW

**Comment by phantomas1234** [comment]
Shouldn't there be a `finally` somewhere in changes?

**Comment by phantomas1234** [comment]
Should merge anyhow if it breaks memote online currently

File: memote/suite/cli/runner.py [hunk 1]
```diff
@@ -676,12 +676,38 @@ def _setup_travis_ci(gh_repo_name, auth_token, repo_access_token):
...
+                "been enabled on Travis CI. Congrats!")
+            break
+        else:
+            LOGGER.info("Still activating...")
+            sleep(0.5)
+    if not activated:
+        LOGGER.critical("Unable to enable automatic testing on Travis CI! "
+                        "Delete all tokens belonging to this repo at "
+                        "https://github.com/settings/tokens then try running "
+                        "`memote new` again. If this fails yet again, please "
```
Comment (Midnighter) [inline memote/suite/cli/runner.py]:
```suggestion
                        "`memote online` again. If this fails yet again, please "
```

File: memote/suite/cli/runner.py [hunk 2]
```diff
@@ -676,12 +676,38 @@ def _setup_travis_ci(gh_repo_name, auth_token, repo_access_token):
...
+            LOGGER.info("Check if activation {} on Travis CI "
+                        "was successful".format(gh_repo_name))
+            response = requests.get(
+                "https://api.travis-ci.org/repo/{}".format(url_safe_repo_name),
+                headers=headers
+            )
+        except HTTPError as error:
+            LOGGER.critical(
+                "An error occurred. Please refer to the following error "
+                "code for further information:".format(str(error)))
```
Comment (Midnighter) [inline memote/suite/cli/runner.py]:
```suggestion
                "message for further information: {}".format(str(error)))
```

Comment (ChristianLieven) [reply memote/suite/cli/runner.py]: I tried it out with the default settings and it works at least for me.

**Pull request merged**